<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputInfo</class>
 <widget class="QWidget" name="InputInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入行数：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editRow">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入列数：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCol">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
